.gitignore

```diff
@@ -4,3 +4,6 @@ yarn-error.log
 .env
 *.iml
 .idea
+types/ethers-contracts
+dist
```

Comment:
add slashes in front of the ignore folder otherwise it will apply for subfolders:
=> /dist
=> /build

Reply:
didnt know that. thanks!